pdfminer/pdfdocument.py

```diff
@@ -895,6 +901,105 @@ def search(entry: object, level: int
             return
         return search(self.catalog['Outlines'], 0)
 
+    def get_page_labels(
+        self,
+        total_pages: Optional[int] = None
+    ) -> Iterator[str]:
+        """
+        Generate page label strings for the PDF document.
+
+        If the document includes page labels, generates strings, one per page.
+        If not, raises PDFNoPageLabels.
+
+        If total_pages is specified, it is used to determine when the iteration
+        stops; if not, the iteration is unbounded.
```

Comment:
I prefer to remove the `total_pages` label, since the Page label object itself is also unbouned. Limiting the number of page labels is never used (if I'm not mistaken).

Reply:
Good catch; it was only used in a test. I've removed it.